docs/ehrql-new-tutorial-1a.md

```diff
@@ -1,56 +1,36 @@
-# ehrQL tutorial: A minimal dataset definition
+# ehrQL tutorial Part 1: Minimal dataset definition
+By the end of this tutorial, you should able:
 
----8<-- 'includes/data-builder-danger-header.md'
+* to write a very simple dataset definition
+* to run that dataset definition with Data Builder
 
-## Example dataset definition 1a: A minimal dataset definition
+## Full Example
 
-### Learning objectives
-
-By the end of this tutorial, you should know how to:
-
-* Write a very simple dataset definition
-* Run that dataset definition with Data Builder
-
-### The dataset definition we will work with
-
-This is a minimal, but still valid, dataset definition:
-
-!!! todo
-
-    May need to fix up how this code is included.
-    Can we have code annotations?
-    What's the best way of jumping between input, output and dataset
-    definition? Maybe that's a case for having one example per tutorial
-    page?
+We start with a minimal dataset definition. This
+finds the patients whose year of birth is 2000 or later.
```

Comment:
suggestion: Could we preserve the semantic linebreaks in the Markdown?

For example:

```suggestion
We start with a minimal dataset definition.
This finds the patients whose year of birth is 2000 or later.
```

Reply:
changes on next commit